index.xml

```diff
@@ -13749,10 +13769,11 @@ C implements A;</x:codeblock>
               <li>If the object <a class='dfnref' href='#dfn-support-indexed-properties'>supports indexed properties</a>, then
               the object’s <a class='dfnref' href='#dfn-supported-property-indices'>supported property indices</a> are
               enumerated first, in numerical order.</li>
-              <li>If the object <a class='dfnref' href='#dfn-support-named-properties'>supports named properties</a>, then
+              <li>If the object <a class='dfnref' href='#dfn-support-named-properties'>supports named properties</a> and doesn't implement an <a class='dfnref' href='#dfn-interface'>interface</a> with the
+              <a class='xattr' href='#LegacyUnenumerableNamedProperties'>[[LegacyUnenumerableNamedProperties]]</a>
+              <a class='dfnref' href='#dfn-extended-attribute'>extended attribute</a>, then
```

Comment:
How carefully has the enumerability (or not) of this stuff been audited?  I thought it was pretty thorough, but the `Window` thing above makes me worried now...

Again, this has the weirdness where the named getter and the extended attribute could be on different interfaces.  Maybe the definition of the extended attribute should require that it only appears on interfaces with a named getter?